app.py

```diff
@@ -75,6 +75,27 @@ def skill():
         return jsonify({})
 
     if request.method == 'POST':
-        return jsonify({})
+        return add_skill()
 
     return jsonify({})
+
+
+def add_skill():
+    
+    '''
+     Add a skill using POST method
+     
+    '''
+    req = request.get_json()
+
+    required_fields = {"name":"string", "proficiency":"string", "logo":"string"}
```

Comment:
Here you are manually repeating the fields from the Skill dataclass, which might be error prone. In general, it is a good idea to have only one source of information and always refer to it. I think you can achieve this with the [`.fields`](https://docs.python.org/3/library/dataclasses.html#dataclasses.fields) method of dataclasses. Can you look into that and let me know if this is feasible? :)

Reply:
I am thinking of using a list comprehension to get the required fields from the skill dataclass. Something akin to 
  > required_fields = [field. name for field in fields(Skill)]. I have tried it, and it works. Will this do?

Reply:
On the Errors, I have managed to fix the pylint error and some of the pytest test cases, notably _test_skill() & test_client()._ The remaining 2 test cases are associated with education and experience whose functionalities are yet to be implemented.